secure/key-manager/Cargo.toml

```diff
@@ -14,7 +14,7 @@ once_cell = "1.4.0"
 serde = { version = "1.0.110", features = ["rc"], default-features = false }
 thiserror = "1.0"
 
-libra-config = { path = "../../config", version = "0.1.0", features = ["fuzzing"]}
```

Comment:
@sunshowers not sure if their's lint for this, but if their is, it may be that the [features] section that already had a fuzzer section listing this crate hid it from your lint.